### ISSUE TITLE
Add the option to have an interval when opening multiple articles at once

### ISSUE THREAD

**Issue by Gojkira** [newsboat/newsboat#2584]
Newsboat 2.33.0 

If there are many unread articles in a feed and the user wants to open them in a browser, it's easiest to open them all at once. However this comes with the side effect of potentially being blocked by the domain for having too many requests at once. It would be nice to have a feature that when executing open-all-unread-in-browser-and-mark-read that instead of opening all articles at once, the user can change the config file to add an x second delay when opening each article to avoid being blocked or having to solve captchas. This would be more convenient than forcing  the user to open all articles individually.

**Comment by der-lyse** [comment]
I reckon you could use a macro with a a custom script that delays opening URLs, something along those lines:

```
macro x set browser "echo %u >> /tmp/urls"; \
    open-all-unread-in-browser-and-mark-read; \
    set browser "~/bin/open-delayed.sh &"; \
    open-in-browser-noninteractively; \
    set browser "firefox"
```

_~/bin/open-delayed.sh_:
```bash
#!/bin/sh

n=0
while read -r url; do
	n=$((n + 1))
	if [ $((n % 3)) -eq 0 ]; then
		sleep 3
	fi
	firefox "$url"
done < /tmp/urls

rm /tmp/urls
```

However, I currently get an error in Newsboat that the browser aborted with exit code 127. Not sure what's going on here. You might figure that out and get it fully working, though.

**Comment by Gojkira** [comment]
This works perfectly for me. Thank you.
